downstream/modules/platform/con-operator-upgrade-prereq.adoc

```diff
@@ -5,6 +5,9 @@
 
 To upgrade to a newer version of {OperatorPlatformNameShort}, you must:
 
-* Create AutomationControllerBackup and AutomationHubBackup objects. For help with this see link:{URLOperatorBackup}[{TitleOperatorBackup}]
+* Ensure your system meets the system requirements detailed in the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/tested_deployment_models/index#ocp-topologies[Operator topologies] section of the _{TitleTopologies}_ guide
```

Comment:
In  my build, this link doesn't go to the section of the topologies doc - it goes to the top of the topologies doc.